thunderbird/processes/wps_generate_prsn.py

```diff
@@ -0,0 +1,231 @@
+# Processor imports
+from pywps import (
+    Process,
+    LiteralInput,
+    ComplexInput,
+    LiteralOutput,
+    ComplexOutput,
+    FORMATS,
+    Format,
+)
+from pywps.app.Common import Metadata
+from pywps.inout.outputs import MetaLink, MetaLink4, MetaFile
+from pywps.inout.literaltypes import AllowedValue
+from pywps.validator.mode import MODE
+from pywps.app.exceptions import ProcessError
+from requests import head
+from requests.exceptions import ConnectionError, MissingSchema, InvalidSchema
+
+# Tool imports
+from nchelpers import CFDataset
+from dp.generate_prsn import generate_prsn_file
+from dp.generate_prsn import dry_run as dry_run_info
+import dp
+
+# Library imports
+import logging
+import os
```

Comment:
Should I only remove the unused imports or also fix the other `flake8` issues? There are some in `test_wps_generate_climos.py` (mostly missing whitespaces after commas) and some `line too long` errors in `common.py` because of the length of the THREDDS URLs that I'm not sure how I'd deal with.

Reply:
We probably should. If you run `python black .` it will fix some of the those errors for you. I believe there is a line-length flag as well to make `flake8` happy. It would be worthwhile to add `flake8` to our `pytest`-ing and CI. Would you be comfortable integrating that?

Reply:
How do I add flake8 to the CI?